velox/exec/Spiller.h

```diff
@@ -205,9 +210,9 @@ class Spiller {
 
   // Define the spiller stats.
   struct Stats {
-    uint64_t spilledBytes = 0;
-    uint64_t spilledRows = 0;
-    uint32_t spilledPartitions = 0;
+    uint64_t spilledBytes{0};
+    uint64_t spilledRows{0};
+    uint32_t spilledPartitions{0};
```

Comment:
Would you document this value. Is it total number of partitions X number of drivers?

Reply:
Commented. Thanks!